src/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Rust Dublin Community
+description: This is a page for the Rust community in Dublin.
+layout: page.njk
+---
+
+<link rel="stylesheet" href="C:\Users\Cillian\Documents\GitHub\website\src\fonts" type="text/css" charset="utf-8" />
```

Comment:
This shouldn't be a hard link to a file on your machine. You should create a file in `src/css/stylesheet.css` 

And add the following content to it
```
@font-face {
   font-family: alfa_slab;
   src: url(/fonts/alfaslab/alfaslabone-regular-webfont.woff);
}
@font-face {
   font-family: fira_sans;
   src: url(/fonts/firasans/firasans-regular-webfont.woff);
}
```
You should then update your font-families in the tags to match the ones defined above.